en-devonfw.dict

```diff
@@ -1206,3 +1201,30 @@ jre
 propertymerge
 autocompletion
 Chocolatey
+
+FreeMarker
+JSON
+JUnit 
+Ionic
+IONIC
```

Comment:
Which one is the correct spelling Ionic or IONIC? We should only have one.

Reply:
Removed IONIC and just added Ionic to dict